Sources/Fluent/Pivot/Pivot.swift

```diff
@@ -0,0 +1,57 @@
+public final class Pivot<
+    L: Entity,
+    R: Entity
+>: PivotProtocol, Entity {
+    public typealias Left = L
+    public typealias Right = R
+
+    public var exists = false
+
+    public static var entity: String {
+        if Left.name < Right.name {
+            return "\(Left.name)_\(Right.name)"
```

Comment:
If you have the tables

```
users
pets
```

The pivot should be 
```
user_pet
```

(Notice they are both not plural)